src/item.rs

```diff
@@ -138,7 +137,7 @@ impl Discriminant {
 						}
 					}
 				} else {
-					return Err(Error::repr_parse(attr.span()));
+					unreachable!("found invalid `repr` attribute")
```

Comment:
did we check, that rustc detects that before executing the proc_macros?

Reply:
I did, yeah. Our proc-macro isn't even executed in that case.